docker/quickstart/docker-compose-without-neo4j-m1.quickstart.yml

```diff
@@ -0,0 +1,161 @@
+networks:
```

Comment:
Quickstart files should only be created through the python code that generates these files. Can you add logic to this file to add this m1 file? https://github.com/linkedin/datahub/blob/9cfc16bc85688b999a1fe98944ea8dedf5d66246/docker/quickstart/generate_docker_quickstart.py